test/unit/org/apache/cassandra/service/accord/AccordConfigurationServiceTest.java

```diff
@@ -218,7 +218,8 @@ public AsyncResult<Void> onTopologyUpdate(Topology topology, boolean isLoad, boo
         }
         finally
         {
-            journal.shutdown();
+            if (journal != null)
```

Comment:
not directly a bug fix... when i broke one serializer the error was lost due to this NPE.  So made it so we don't NPE and get the original error